### PR TITLE
HCCDOC3138:Add Insights Runtime Extractor TP feature to Insights Operator release notes

### PR DESCRIPTION
**Pull request by michelle-purcell**
OCP 4.18 introduces a new feature for workload data collection called the Insights Runtime Extractor.
The feature is releases as a **Technology Preview** for OCP 4.18.

Version(s):

4.18 only

Issue:

https://issues.redhat.com/browse/HCCDOC-3138
and
https://issues.redhat.com/browse/HCCDOC-3143
https://issues.redhat.com/browse/OCPSTRAT-1694

Link to docs preview:

https://84967--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-18-release-notes.html#ocp-release-notes-insights-operator-enhancements_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-18-release-notes.adoc [hunk 1]
```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+==== Insights Runtime Extractor (Technology Preview)
+
+In {product-title} 4.18, the Insights Operator introduces a new workload data collection feature, _Insights Runtime Extractor_ (`InsightsRuntimeExtractor`), to help Red{nbsp}Hat better understand the workload of your containers.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-18-release-notes.adoc:414]: Not sure what type of thing the `InsightsRuntimeExtractor` in backticks is - a CRD? Does that actually need mention in a RN blurb at all, or does it make sense to just use the "Insights Runtime Extractor" feature name here and throughout the RN text?
```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers.
```

File: release_notes/ocp-4-18-release-notes.adoc [hunk 2]
```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
...
+Available as a Technology Preview, the `InsightsRuntimeExtractor` feature gathers and sends runtime workload data to Red{nbsp}Hat.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-18-release-notes.adoc:415]:
```suggestion
Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. 
```

File: release_notes/ocp-4-18-release-notes.adoc [hunk 3]
```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+==== Insights Runtime Extractor (Technology Preview)
+
+In {product-title} 4.18, the Insights Operator introduces a new workload data collection feature, _Insights Runtime Extractor_ (`InsightsRuntimeExtractor`), to help Red{nbsp}Hat better understand the workload of your containers.
+Available as a Technology Preview, the `InsightsRuntimeExtractor` feature gathers and sends runtime workload data to Red{nbsp}Hat. 
+Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.
+For more information about how to enable the feature, see _Enabling features using feature gates_.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-18-release-notes.adoc:417]: In addition to the suggestions above, consider breaking this up and adding a link to "Enabling features using feature gates":

```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers.

Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. 
Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.

For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

File: release_notes/ocp-4-18-release-notes.adoc [hunk 4]
```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+==== Insights Runtime Extractor (Technology Preview)
```
Comment (jeana-redhat) [inline release_notes/ocp-4-18-release-notes.adoc:412]: This requires an id
```suggestion
[id="ocp-release-notes-insights-operator-runtime-extractor_{context}"]
==== Insights Runtime Extractor (Technology Preview)
```

**Review by jeana-redhat — COMMENTED** [review]
Some things to consider here

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-18-release-notes.adoc (hunk 3 above)
Comment (mburke5678) [inline release_notes/ocp-4-18-release-notes.adoc:417]: @michelle-purcell Do we know for certain that the user needs to enable the TechPreviewNoUpgrade feature set to use this feature? This link goes to a procedure to do just that, which has significant consequences for the cluster.

File: release_notes/ocp-4-18-release-notes.adoc (hunk 1 above)
Comment (michelle-purcell) [reply release_notes/ocp-4-18-release-notes.adoc]: It's the actual name of the feature resource that customers enable through feature gates. 
We also add and refer this feature to the 'Enabling feature gates' topic too, and use this same format, similar to other data collection features/resources that are in Tech preview.

File: release_notes/ocp-4-18-release-notes.adoc (hunk 3 above)
Comment (michelle-purcell) [reply release_notes/ocp-4-18-release-notes.adoc]: Done.  
Good idea. Note that, previous peer review suggestions requested that my xrefs were removed from release notes and added to the Additional Information section of the assembly instead.

**Comment by michelle-purcell** [comment]
@jeana-redhat - Thanks so much for your thorough review. All comments are now resolved. (Apologies for the slow response in processing. I was on PTO. )

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM; merging.

**Comment by michelle-purcell** [comment]
@sheriff-rh - Thank you!

**Pull request merged**